docs/quickstarts/ratify-on-azure.md

```diff
@@ -491,26 +478,25 @@ Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepoli
     apiVersion: config.ratify.deislabs.io/v1beta1
     kind: Verifier
     metadata:
-    name: verifier-notation
+      name: verifier-notation
     spec:
-    name: notation
-    artifactTypes: application/vnd.cncf.notary.signature
-    parameters:
+      name: notation
+      artifactTypes: application/vnd.cncf.notary.signature
+      parameters:
         verificationCertStores:
```

Comment:
`verificationCertStores` is recommended to add the truststore type, like the `ca` in this case. https://ratify.dev/docs/next/plugins/verifier/notation#kubernetes

Reply:
@binbin-li 
When I revert this change to include `ca`, it doesn't successfully apply the yaml file:

kubectl describe Verifier verifier-notation
```
Name:         verifier-notation
Namespace:    
Labels:       <none>
Annotations:  helm.sh/hook: pre-install,pre-upgrade
              helm.sh/hook-weight: 5
API Version:  config.ratify.deislabs.io/v1beta1
Kind:         Verifier
Metadata:
  Creation Timestamp:  2024-08-27T00:06:43Z
  Generation:          2
  Resource Version:    1874082
  UID:                 13626af0-98f6-4de3-9ae2-99a1907a58c1
Spec:
  Artifact Types:  application/vnd.cncf.notary.signature
  Name:            notation
  Parameters:
    Trust Policy Doc:
      Trust Policies:
        Name:  default
        Registry Scopes:
          *
        Signature Verification:
          Level:  strict
        Trust Stores:
          ca:certs
        Trusted Identities:
          x509.subject: CN=wabbit-networks.io,O=Notation,L=Seattle,ST=WA,C=US
      Version:  1.0
    Verification Cert Stores:
      Ca:
        Certs:
          keymanagementprovider-akv
      Certs:
        ratify-notation-inline-cert-0
  Version:  1.0.0
Status:
  Brieferror:  Error: config invalid, Code: C...
  Error:       Error: config invalid, Code: CONFIG_INVALID, Plugin Name: verifier-notation, Component Type: verifier
  Issuccess:   false
Events:        <none>

```

I checked one sample notation config file here: ./conffig/samples/clustered/verifier/config-v1alpha1-verifier-notation.yaml and it also didn't include ca in verificationcertstores.

Reply:
curious if it should `ca` instead of `Ca`?

Reply:
No worries. I will answer this.

Reply:
You can retry after the https://github.com/ratify-project/ratify/pull/1758 get merged